projects/saturn/src/main/java/io/fairspace/saturn/services/views/ViewStoreReader.java

```diff
@@ -445,10 +441,28 @@ List<Map<String, Set<ValueDTO>>> retrieveJoinTableRows(
         return rows;
     }
 
+    private PreparedStatement getQuery(String view, View.JoinView joinView, Table joinedTable, Table joinTable,
```

Comment:
hm-m, out of context the method name is not clear, I would add 'join' to the naming. What do you think?